ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactionsMessageProcessor.java

```diff
@@ -96,7 +98,14 @@ void processNewPooledTransactionHashesMessage(
   private void processNewPooledTransactionHashesMessage(
       final EthPeer peer, final NewPooledTransactionHashesMessage transactionsMessage) {
     try {
-      LOG.trace("Received pooled transaction hashes message from {}", peer);
+      final List<Hash> incomingTransactionHashes = transactionsMessage.pendingTransactions();
+
+      traceLambda(
+          LOG,
+          "Received pooled transaction hashes message from {}, incoming hashes {}, incoming list {}",
+          peer::toString,
+          incomingTransactionHashes::size,
+          incomingTransactionHashes::toString);
 
       transactionTracker.markTransactionsHashesAsSeen(
           peer, transactionsMessage.pendingTransactions());
```

Comment:
Instead of `transactionsMessage.pendingTransactions()` we could use `incomingTransactionHashes` directly.

Reply:
right, I will incorporate this in the next change that I am working on to improve eth/66 support